sources/packages/api/src-sql/StudentFiles/Add-meta-data-and-file-origin.sql

```diff
@@ -0,0 +1,23 @@
+-- set default
+ALTER TABLE
+  sims.student_files
+ALTER COLUMN
+  group_name
+SET
+  DEFAULT 'Student Temporary File';
+
+-- add column
+ALTER TABLE
+  sims.student_files
+ADD
+  COLUMN IF NOT EXISTS file_origin sims.file_origin_type NOT NULL DEFAULT 'Temporary';
+
+COMMENT ON COLUMN sims.student_files.file_origin IS 'File original originated from like Application or Student uploader form. If its Temporary, then the file is uploaded but the file uploaded form is not submitted, when the form is submitted, the file origin is updated from Temporary to the respective file_origin_type';
```

Comment:
Really nice comment. I would adjust a bit as below, just as a suggestion.

File originated from, for instance, an Application or Student uploader form. If it's Temporary, then the file is uploaded but the file uploaded form is not submitted yet. When the form is submitted, the file origin is updated from Temporary to the respective file_origin_type.